app/controllers/people_controller.rb

```diff
@@ -0,0 +1,37 @@
+class PeopleController < ApplicationController
+
+  def index
+    # grabs all the people
+    @people = Person.all
+    @tasks = Task.all
+  end
+
+  def show
+    find_person
+  end
+
+  def tasks
+    find_person
+    find_tasks
+  end
+
+  def find_tasks
+    tasks = Task.all
+    @their_tasks = tasks.where(person_id: @person.id)
+  end
+
+
+  private
+    # permissable parameters should be in a private method,
+    # ie not accessible outside the class TaskController
+    def find_person
```

Comment:
I really like that you moved this to a private method since you were using this logic more than once!